storybookjs/storybook
# Maintenance: Add TypeScript plugin for Yarn

Issue:

## What I did

Added the TypeScript plugin for Yarn to automatically install `@types` packages when adding dependencies without their own types.
https://github.com/yarnpkg/berry/blob/e81dc0d29bb2f41818d9c5c1c74bab1406fb979b/packages/plugin-typescript/README.md 

## How to test

Run the following command and observe that `lodash` and `@types/lodash` is installed into the workspace

```
yarn workspace @storybook/preact add lodash@4
```
